CONTRIBUTING.md

```diff
@@ -35,3 +35,50 @@ We strive to follow this [recommendation](https://chris.beams.io/posts/git-commi
 We add this additional rule:
 
 - Do not rely on GitHub issue reference numbers in commit messages, as we have no guarantee the host system and its autolinking will be stable in time. Make sure the context is self-explanatory. If an external reference is given, use its full URL.
+
+## Add a contributor
+
+Some contributors may wish to be listed as a contributor on the [README](./README.md#contributors) file and the Open Terms Archive [website](https://opentermsarchive.org/about).
+
+### For contributors with a GitHub account
+
+This is made possible by the use of [All Contributors](https://allcontributors.org/) bot that allows you to add a contributor with a comment on Issue or pull request, without writing code. To do this you need to open an issue on this repository and write a command like `@all-contributors please add @<username> for <contributions>`. See the complete [documentation](https://allcontributors.org/docs/en/bot/usage) for more details.
+
+### For contributors who do not have a GitHub account
+
+1. Edit the `.all-contributorsrc` file and use the following format to add a contributor:
+
+```json
+{
+  ...,
+  "contributors": [
+    {
+      "name": "<contributor_name>",
+      "profile": "<contributor_website>",
+      "avatar_url": "https://opentermsarchive.org/images/contributors/<slugify-contributor-name>.jpg",
+      "contributions": ["<type_of_contribution>"]
+    }
+  ]
+}
+```
+
+2. Add a picture to `public/images/contributors` folder which respects the following constraints:
+
+- square format, 420x420px
+- in optimised `jpg`
+- slugify the filename, for exemple: `contributor-name.jpg`
+
+3. Generate the contributors table in the readme file:
+
+- install cli: `npm install all-contributors-cli --save-dev`
+- generate: `npx all-contributors generate`
+
+### How to add a type of contribution to a contributor
+
+You can do this by editing the `.all-contributorsrc` file or by using the bot with `@all-contributors please add @<already_listed_contributor> for <new_contribution_type>`
```

Comment:
This is not clear without an understanding of how All Contributors works: we need to document how a contribution type is mandatory, and link to the list of types.

Reply:
> we need to document how a contribution type is mandatory

I don't understand what needs to be explained. 

I've added a link to the list of types.

Reply:
There is no explanation in the CONTRIBUTING that a “contributor” needs to have a “contribution type”, nor a reference of which ones we use and the interpretation we give to it 🙂